src/org/jitsi/util/RTPUtils.java

```diff
@@ -44,6 +44,7 @@ else if (diff > 1<<15)
      * (modulo 2^16).
      * @return result of the subtraction of one RTP sequence number from another
      * (modulo 2^16).
+     * FIXME(brian): why do we have this and sequenceNumberDiff?
```

Comment:
sequenceNumberDiff takes into account the possibility of a sequence number wrap-around. substractNumber performs a substraction mod 2^16. substractNumber is poorly named :-)